cmd/crank/beta/convert/pipelinecomposition/converter.go

```diff
@@ -147,15 +149,116 @@ func processFunctionInput(input *Input) *runtime.RawExtension {
 		"apiVersion":  "pt.fn.crossplane.io/v1beta1",
 		"kind":        "Resources",
 		"environment": processedInput.Environment.DeepCopy(),
```

Comment:
we should also migrate patches in the environment 😅

Reply:
ah, good catch @phisco! 🙇‍♂️ 

@padlar I can add a commit to this PR to handle the environment patches too 😉

Reply:
@phisco @padlar I pushed a new commit that implements migration for environment patches too. It also added more unit test coverage for all the `MigratePatchPoliciesIn*` functions.

Please take a look when you can! 🙇‍♂️